### ISSUE TITLE
Training, testing, and validation set

### ISSUE THREAD

**Issue by sensationTI** [AlexeyAB/darknet#6413]
All the training data go into the `\obj` folder and all validation data go into the`\val` folder. Where is the testing folder?

**Comment by stephanecharette** [comment]
Note that the images, labels, data files, cfg, etc don't have to be in the darknet directory.  You can setup your project to use a directory structure which is outside of the Darknet source, and you call darknet directly and point to the location where it needs to run and find all the necessary files.  I have an example of that in this tutorial:  https://www.ccoderun.ca/programming/2020-03-07_Darknet/
